integration/tests/params.go

```diff
@@ -54,6 +54,7 @@ type Params struct {
 // on the region field.
 func (p *Params) SetRegion(region string) {
 	p.Region = region
+	p.Region = "ap-northeast-2"
```

Comment:
what's going on here?

Reply:
going to run the integration tests in github actions, pinning it to this region temporarily

Reply:
hmmm alright, I'll allow it